tests/api/test_app.py

```diff
@@ -341,24 +806,30 @@ def test_app_with_access_to_resources(
       authToken
       id
     }
-    errors{
+    appErrors{
       field
       message
+      code
     }
   }
 }
 """
 
 
-def test_app_token_create(permission_manage_apps, staff_api_client, staff_user):
+def test_app_token_create(
+    permission_manage_apps, staff_api_client, staff_user, permission_manage_orders
+):
 
     app = App.objects.create(name="New_sa")
```

Comment:
Could you also rename this to `app` instead of shortcut form `ServiceAccount`. It seems that I missed that :c